app/views/admin/registration_periods/show.html.haml

```diff
@@ -8,29 +8,38 @@
   .row
     .col-md-8
       %dl.dl-horizontal
-        %dt
-          Start Date
-        %dd
-          = @registration_period.start_date
-        %dt
-          End Date
-        %dd
-          = @registration_period.end_date
+        %dt Start Date
+        %dd= @registration_period.start_date
+        %dt End Date
+        %dd= @registration_period.end_date
+        %dt Ticket required?
+        %dd= @conference.tickets.for_registration.any? ? 'Yes' : 'No'
+
+.row
+  .col-md-12
+    %p
+      If you would like to require a ticket purchase for registration, at least
+      one
+      = link_to 'ticket', admin_conference_tickets_path(@conference)
+      must be set as a registration ticket, and then the registration process
+      will not process without the purchase (or selection of a free) single
```

Comment:
What will the process not process? 
I assume we mean that a ticket must be set as a registration ticket, so that the registration process proceeds with the purchase of a registration ticket, or the selection of a free one?

Reply:
Have you ever read something so many times that you can't actually read it anymore? :/

Reply:
I reworded in bedb6fad . Thanks for catching that... I would have read past it 100 times ;-)